client/web/privacy/src/components/ConnectionMap.vue

```diff
@@ -1,61 +1,47 @@
 <template>
-  <l-map
-    :zoom="zoom"
+  <c-map
```

Comment:
If you compare how this looks locally and on the previous version you'll see the styles are a bit different